src/BUILD

```diff
@@ -412,7 +411,8 @@ cc_binary(
     ],
     deps = [
         "@google_mediapipe//mediapipe/examples/desktop:simple_run_graph_main",
-        "@mediapipe_calculators//:mediapipe_calculators",
+        #"@mediapipe_calculators//:mediapipe_calculators",
```

Comment:
Why it is commented out?

Reply:
Currently one required calculator is commented out in mediapipe, we can enable it later and chenge it to this dependency.